cypress/integration/account_settings/theme_color/custom_colors_sidebarStyles_spec.js

```diff
@@ -0,0 +1,441 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// **************************************************************
+
+describe('Account Settings > Display > Theme Colors > Custom Theme > Sidebar Styles', () => {
+    before(() => {
+        // 1. Go to Account Settings with "user-1"
+        cy.toAccountSettingsModal('user-1');
+    });
+
```

Comment:
Add new command that reverts the changes to theme setting and use it here for `after` hook so that the last test below of `it('should revert any color changes made', () => {...}` can be removed.

Reply:
Added a new command to revert theme color changes called `defaultTheme`, created an after `after` hook. 

`after(() => {
        cy.defaultTheme('user-1');
    });`